app/Http/Controllers/ResponseController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Response as ResponseQuestion;
+use Illuminate\Http\{Request, JsonResponse};
+use Illuminate\Support\Facades\{DB, URL, Validator, Response};
+use Illuminate\Http\Resources\Json\AnonymousResourceCollection;
+use App\Http\Resources\Response as ResponseResource;
+use Illuminate\Validation\Rule;
+
+class ResponseController extends Controller
+{
+
+
+    /**
+     * @return AnonymousResourceCollection
+     */
+    public function index(): AnonymousResourceCollection
+    {
+        return ResponseResource::collection(ResponseQuestion::paginate(10));
+    }
+
+    /**
+     * @param int $id
+     * @return ResponseResource
+     */
+    public function show(int $id): ResponseResource
+    {
+        return new ResponseResource(ResponseQuestion::find($id));
+    }
+
+    /**
+     * @param Request $request
+     * @return JsonResponse|\Illuminate\Support\MessageBag
+     * @throws \Exception
+     */
+    public function store(Request $request)
+    {
+        $validation = Validator::make($request->all(), [
+            'text' => 'required|string|min:1|max:255',
+            'question_id' => 'integer',
+        ]);
+
+        if ($validation->fails())
+            return $validation->errors();
+
+
+        $dataToInsert = $request->only(['text', 'question_id']);
```

Comment:
Instead of add `$request->only(['text', 'question_id']);` to a variable, you should add `$request->only(['text', 'question_id']);` in the method parameter `create`